Hometask 2

emervlad opened the pull request:
Door и Light Processor'ы в прошлом дз были перепутаны в названиях.

emervlad committed:
Add files via upload

a-ivanov commented on new/SmartHome.java:
```diff
@@ -21,4 +21,10 @@ public void addRoom(Room room) {
     public Collection<Room> getRooms() {
```
Удалите.

a-ivanov reviewed (changes requested):
Удалите все методы с доступом к внутренним объектам дома.

emervlad committed:
Add files via upload

По поводу первого ДЗ: почти все ошибки попытался исправить во втором ДЗ, кроме O и D принципов в классе EventProcessor, его сейчас исправил.
Во втором помимо геттеров комнат и пр. убрал также и геттеры, которые дают имя, id, и тп., поскольку нужда в них также отпала
